main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/gptscript-ai/datasets/pkg/dataset"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("missing argument")
```

Comment:
Can we add what we expect the argument to be? I see the switch below, but maybe something like `missing action as first argument` would be more descriptive so the LLM can recover?

Reply:
done